src/main/resources/i18n/messages.properties

```diff
@@ -3,3 +3,19 @@ error.title=Your request cannot be processed
 error.subtitle=Sorry, an error has occurred.
 error.status=Status:
 error.message=Message:
+
```

Comment:
Are you sure these properties are used in our gate code?

Reply:
done.